server/models/schema.sql

```diff
@@ -2182,10 +2182,31 @@ CREATE TABLE `fee_center` (
   `label` VARCHAR(100) NOT NULL,
   `is_principal` TINYINT(1) UNSIGNED DEFAULT 0,
   `project_id` SMALLINT(5) UNSIGNED NULL,
+  `step_order` SMALLINT(5) NOT NULL DEFAULT 100,
+  `default_fee_center_index_id` mediumINT(8) UNSIGNED NULL,
```

Comment:
I assume this is the default ordering?

Reply:
Yes, it is supposed to be defined by the `step_order` column